tortoise/backends/sqlite/client.py

```diff
@@ -81,6 +82,7 @@ async def create_connection(self, with_db: bool) -> None:
             for pragma, val in self.pragmas.items():
                 cursor = await self._connection.execute(f"PRAGMA {pragma}={val}")
                 await cursor.close()
+            await install_regexp_function(self._connection)
```

Comment:
Seems a bit strange that we are automatically installing new functions in user db just on connection

May be we should add some kind of flag to make it explicit and only in case user need it? 
If we do so, also would be nice to check if it was activated inside function and propagate proper error for user to prompt him to switch that flag

Even though it sounds like unneeded extra steps - it seems reasonable to not modify user db without explicit requests to do so